lib/simulation/openfisca/mapping/index.js

```diff
@@ -76,7 +71,7 @@ var applyRessources = function(mesAidesEntity, openFiscaEntity, mappingSchema, s
 
                         // Aggrégats
                         _.keys(periods).forEach(function (agregatKey) {
-                            if (agregatKey != 'anneeFiscalReference' && isPeriodWithin(agregatKey, moment(item.periode, 'YYYY-MM'))) {
+                            if (agregatKey != 'referenceFiscalYear' && isPeriodWithin(agregatKey, moment(item.periode, 'YYYY-MM'))) {
```

Comment:
Isn't “Année fiscale de référence” a domain name? If so, it shouldn't be translated to English.